docs/pages/develop/user-interface/fonts.mdx

```diff
@@ -43,9 +44,47 @@ For reference, the following table provides what formats work on which platforms
 
 ## Use a custom font
 
+There are two ways to use a custom font in your project: you can either embed the font in your native project or load it at runtime. The first approach is recommended as it is simpler and more reliable. The second approach is useful if you want to load the font in Expo Go / without creating a new native build that includes it, or load it from a remote URL.
+
+### 1. Embed the font in your native project
+
+> Note: this `expo-font` config plugin is only available for SDK 50 and higher. If you are using an older SDK, you can [load the font at runtime](#2-load-the-font-at-runtime) instead. 
+
+Install [`expo-font`](/versions/latest/sdk/font/#installation) package and add the config plugin to your `app.json` file. The plugin will embed the font in your native project.
```

Comment:
```suggestion
Install [`expo-font`](/versions/latest/sdk/font/#installation) library and add the config plugin to your [app config](/versions/latest/config/app/) file. The plugin will embed the font in your native project.
```

- Let's prefer library instead of package as per our writing style guide
- Let's link to app.json config and use app config because app.json file can also be app.config.js, etc.